README.md

```diff
@@ -167,6 +167,43 @@ tsc azureIdentityCredentialAdapter.spec.ts --esModuleInterop
 
 Once you have the `.env` file configured and js compiled, run the test simply calling `mocha azureIdentityCredentialAdapter.spec.js --timeout 10000`.
 
+## Golang
+
+### NewAzureIdentityCredentialAdapter
+
+The `NewAzureIdentityCredentialAdapter` function allows you to use all the goodness of `azidentity` in the Azure Management libraries. You can use it in place of `Authorizer` when calling your Azure Management APIs.
+
+To use this type, just import package github.com/jongio/azidext/go/azidentity and using follow command to get package.
```

Comment:
is the import path staying like this? From the comments I saw the the package will be named `azidext` if so I think it might be more conventional for the path to also end in azidext.

Reply:
Good catch, yes the directory name needs to change to match the package/module name.